packages/gatsby/cache-dir/page-renderer.js

```diff
@@ -37,6 +42,7 @@ class PageRenderer extends React.Component {
 }
 
 PageRenderer.propTypes = {
+  params: PropTypes.object.isRequired,
```

Comment:
Do we need it now that `params` are actually added inside the `render` function and not passed in?

Reply:
removed!